.storybook/config.js

```diff
@@ -0,0 +1,9 @@
+import { configure } from '@storybook/react';
+
+// automatically import all files ending in *.stories.js
+const req = require.context( '../stories', true, /\.stories\.js$/ );
```

Comment:
stories should probably be moved somewhere else. maybe components should contain their own stories the same way as they contain their example which is used in devdocs?

Reply:
yeah, i'd like to keep the stories with the components assuming we go with storybook. similar to how we kept examples with components in base calypso.